source/fundamentals/crud/read-write-pref.txt

```diff
@@ -0,0 +1,259 @@
+.. _node-crud-write-read-pref:
+
+===================================
+Modify Execution of CRUD Operations
+===================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: node.js, customize, preferences, replica set, consistency
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to modify the way that the {+driver-long+} executes create, read, update, and delete (CRUD) operations using **write concern**, **read concern**, and **read preference** configurations for replica sets.
+
+You can set write concern, read concern, and read preference options at
+the following levels:
+
+- Client level, which sets the *default for all operation executions*
+  unless overridden
+- Session level
+- Transaction level
+- Database level
+- Collection level
+
+These options allow you to customize the consistency and availability of the data in your replica sets.
+
+Write Concern
+-------------
+
+A write concern describes the level of acknowledgement requested from MongoDB for write operations, such as an insert or update, before the operation is returned as successful. 
+Operations which do not specify an explicit write concern inherit the global default write concern settings.
+
+For more information, see :manual:`Write Concern </reference/write-concern/>`. For more detailed API descriptions, see `API WriteConcern <{+api+}/classes/WriteConcern.html>`__
+
+Constructor
+~~~~~~~~~~~
+
+A ``WriteConcern`` can define all, some or none of its accepted parameters:
+
+.. code-block:: js
+
+   new WriteConcern(w, wtimeoutMS, journal): WriteConcern
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Parameter
+     - Description
+  
+   * - ``w`` *(optional)*
+     - | request acknowledgment that the write operation has propagated to a specified number of mongod instances or to mongod instances with specified tags 
+       | **Type:** `W ***************** <{+api+}/types/W.html>`__
+
+   *  - ``wtimeoutMS`` *(optional)*
+      - | specify a time limit to prevent write operations from blocking indefinitely
+        | **Type:** number
+
+   * - ``journal`` *(optional)*
+     - | request acknowledgment that the write operation has been written to the on-disk journal
+       | **Type:** boolean
```

Comment:
i: for consistency with other docs, i'd capitalize and use third-person verbs

```suggestion
   * - ``w`` *(optional)*
     - | Requests acknowledgment that the write operation has propagated to a specified number of mongod instances or to mongod instances with specified tags 
       | **Type:** `W ***************** <{+api+}/types/W.html>`__

   *  - ``wtimeoutMS`` *(optional)*
      - | Specifies a time limit to prevent write operations from blocking indefinitely
        | **Type:** number

   * - ``journal`` *(optional)*
     - | Requests acknowledgment that the write operation has been written to the on-disk journal
       | **Type:** boolean
```